app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people: dict = {}
 
+    def __init__(self, name: str, age: str) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people: list) -> list[Person]:
+    person_list: list[Person] = []
+    for person in people:
+        # next line initiates the creation of an instance of the class
+        Person(person["name"], person["age"])
```

Comment:
Remove this code is is redundant

Reply:
no, this code is required. Without it, person_list will not be created because Person.people[person["name"]] will not exist